packages/core/src/components/popover/_popover-in-button-group.scss

```diff
@@ -15,7 +15,12 @@
     > .#{$ns}-popover-target:not(:last-child) .#{$ns}-button {
       border-bottom-right-radius: 0;
       border-top-right-radius: 0;
-      margin-right: -$button-border-width;
+    }
+
+    &:not(.#{$ns}-outlined) {
+      > .#{$ns}-popover-target:not(:last-child) .#{$ns}-button:not(.#{$ns}-outlined) {
```

Comment:
There seems to be a lot of duplicate (or at least similar) styles for popovers here and in the other file. We should probably consolidate them and also merge the selectors for popover-target and popover-wrapper, since they should likely apply the exact same styles (the latter only existing for back-compat anyway).

Reply:
Yeah, I noticed this as well, but was hesitant to tackle reorganizing those styles in this PR since it's a more involved change. I'm also not super familiar with the historical use of the popover-wrapper class, so that also made the prospect of that change seem more brittle to me.

Happy to address cleaning these up though. Would you recommend this PR or a FLUP?